release/models/bgp/openconfig-bgp-types.yang

```diff
@@ -361,6 +367,15 @@ module openconfig-bgp-types {
     reference "RFC3765";
   }
 
+  identity GRACEFUL_SHUTDOWN {
+    base BGP_WELL_KNOWN_STD_COMMUNITY;
+    description
+      "An autonomous system which supports the graceful shutdown
+      receiver procedure receiving NLRI tagged with this community
+      will set LOCAL_PREF to a low value for those NLRI.";
```

Comment:
Could add the value of the community (0xFFFF0000), though we haven't done it consistently in all of these community identities.

Reply:
Good idea, I added.